dimagi/utils/management/commands/testproject.py

```diff
@@ -1,30 +1,34 @@
 from __future__ import print_function
 from django.core.management import call_command
-from django.core.management.base import CommandError, BaseCommand
-from optparse import make_option
-import os
+from django.core.management.base import BaseCommand
 import settings
 
 
 class Command(BaseCommand):
-    option_list = (
-    #       make_option('--file', action='store', dest='file', default=None, help='File to upload REQUIRED', type='string'),
-    #       make_option('--url', action='store', dest='url', default=None, help='URL to upload to*', type='string'),
-    make_option('--xmlreporting', action='store_true', dest='xml_reporting', default=False,
-                help='Use xml reporting for build server integration (default=False)'),
-    make_option('--all', action='store_true', dest='test_all', default=False,
-                help='Test ALL apps in project. (default=False)'),
-    )
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--xmlreporting',
+            action='store_true',
+            dest='xml_reporting',
+            default=False,
+            help='Use xml reporting for build server integration (default=False)',
+        )
+        parser.add_argument(
+            '--all',
+            action='store_true',
+            dest='test_all',
+            default=False,
+            help='Test ALL apps in project. (default=False)',
+        )
+
     help = "Test only the relevant apps as defined in the settings file for your project.  Ignore django standard and other third party apps."
```

Comment:
I think the changes of this command having been used is pretty low considering our other test efforts